model/item.rb

```diff
@@ -24,9 +24,9 @@ def author=(author)
     author.add_item(self)
   end
 
-  def label=(label)
+  def add_label(label)
```

Comment:
- The better way to implement a customized setter is to use `def attribute_name=(value)`, and when calling it you can use `object.attribute_name = 'value`. This way you don't need to know the setter method name, You only need to be aware of the attribute name that you want to change.